src/Error/ErrorTransformer.php

```diff
@@ -17,9 +17,9 @@ public function transform(array $errors): array
     {
         return array_map(
             function ($error) {
-                if ($error->identifier === 'offsetAccess.notFound' && preg_match("/Offset '(?<variableName>.*)' (?<error>might not exist|does not exist) on (.*)\./", $error->message, $matches) === 1) {
-                    return $error->withMessage(sprintf("Variable '%s' %s.", $matches['variableName'], $matches['error']));
-                }
+//                if ($error->identifier === 'offsetAccess.notFound' && preg_match("/Offset '(?<variableName>.*)' (?<error>might not exist|does not exist) on (.*)\./", $error->message, $matches) === 1) {
+//                    return $error->withMessage(sprintf("Variable '%s' %s.", $matches['variableName'], $matches['error']));
+//                }
```

Comment:
I have an issue with this because it changes my Offset might not exist / does not exists into Variable does not exists.

It seems like you added this related to the `context` but maybe you should use a stricter regex ? (You can uncomment to see the impact)